src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,48 @@
+.header__nav {
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+  height: 60px;
+}
+
+.nav__list :last-child {
+  margin-right: 0;
+}
+
+.nav__item,
+.nav__link {
+  display: flex;
+  height: 60px;
+  align-items: center;
+  justify-content: center;
+  margin-right: 20px;
+  font: {
+    size: 12px;
+    style: normal;
+    weight: var(--middle-fontweight);
+  }
+  color: var(--color-dark);
+  text: {
+    transform: uppercase;
+    decoration: none;
+  }
+}
+
+// .nav__link {
+//   line-height: 52px;
```

Comment:
same

Reply:
src/styles/blocks/card.scss
  font: {
    weight: var(--middle-fontweight);
    style: normal;
    size: 12px;
  }
 
@aliceblunt8 aliceblunt8 4 hours ago
nesting🔥  Did i do it wrong?

Reply:
no, everything it's good, it's cool you use this syntax;)